### ISSUE TITLE
Render plot correctly

### ISSUE THREAD

**Issue by Tbruno25** [bczsalba/pytermgui#2]
I'm looking to create multiple plot trend lines inside of a tui. 

Another library I'm utilizing produces a string such as
`'⠀⠀⠀⠀⢠⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⡆⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀\n⠀⠀⠀⠀⡸⡇⠀⠀⠀⠀⠀⠀⠀⠀⢠⢳⠀⠀⠀⠀⠀⣧⠀⠀⡤⡄⠀⠀⠀⠀\n⠀⠀⠀⢀⠇⡇⠀⠀⠀⠀⠀⠀⠀⠀⢸⠘⡄⠀⠀⡀⢠⢻⠀⢀⠇⡇⠀⢀⠀⠀\n⡆⠀⢀⠎⠀⢣⠀⠀⠀⠀⠀⠀⠀⠀⡜⠀⡇⠀⠀⡇⢸⢸⠀⢸⠀⢇⠀⢸⠀⠀\n⢣⠀⡸⠀⠀⢸⠀⠀⠀⠀⠀⠀⠀⠀⡇⠀⢇⠀⢰⢇⢸⠈⡆⢸⠀⢸⠀⡿⡀⠀\n⢸⠀⡇⠀⠀⢸⠀⢀⠤⡆⠀⠀⠀⢸⠀⠀⢸⠀⢸⢸⢸⠀⡇⡎⠀⢸⠀⡇⡇⠀\n⠘⣼⠀⠀⠀⢸⠀⢸⠀⢱⠀⠀⠀⡎⠀⠀⢸⠀⢸⢸⢸⠀⡇⡇⠀⢸⢠⠃⡇⢀\n⠀⠇⠀⠀⠀⢸⠀⡎⠀⠘⡄⠀⢸⠀⠀⠀⠘⡄⡜⢸⢸⠀⢣⠇⠀⢸⢸⠀⡇⢸\n⠀⠀⠀⠀⠀⢸⠀⡇⠀⠀⡇⠀⡇⠀⠀⠀⠀⡇⡇⠸⡇⠀⢸⠀⠀⠀⣿⠀⢣⢸\n⠀⠀⠀⠀⠀⢸⠀⡇⠀⠀⢸⢰⠁⠀⠀⠀⠀⡇⡇⠀⡇⠀⠀⠀⠀⠀⡇⠀⢸⢸\n⠀⠀⠀⠀⠀⠸⣰⠁⠀⠀⠀⠇⠀⠀⠀⠀⠀⢳⠁⠀⠇⠀⠀⠀⠀⠀⠁⠀⢸⢸\n⠀⠀⠀⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⡸\n⠀⠀⠀⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⡇\n⠀⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡇\n⠀⠀⠀⠀⠀⠀⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠃'`

When printed from the python interpreter this formats correctly

`⠀⠀⠀⠀⢠⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⡆⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⡸⡇⠀⠀⠀⠀⠀⠀⠀⠀⢠⢳⠀⠀⠀⠀⠀⣧⠀⠀⡤⡄⠀⠀⠀⠀
⠀⠀⠀⢀⠇⡇⠀⠀⠀⠀⠀⠀⠀⠀⢸⠘⡄⠀⠀⡀⢠⢻⠀⢀⠇⡇⠀⢀⠀⠀
⡆⠀⢀⠎⠀⢣⠀⠀⠀⠀⠀⠀⠀⠀⡜⠀⡇⠀⠀⡇⢸⢸⠀⢸⠀⢇⠀⢸⠀⠀
⢣⠀⡸⠀⠀⢸⠀⠀⠀⠀⠀⠀⠀⠀⡇⠀⢇⠀⢰⢇⢸⠈⡆⢸⠀⢸⠀⡿⡀⠀
⢸⠀⡇⠀⠀⢸⠀⢀⠤⡆⠀⠀⠀⢸⠀⠀⢸⠀⢸⢸⢸⠀⡇⡎⠀⢸⠀⡇⡇⠀
⠘⣼⠀⠀⠀⢸⠀⢸⠀⢱⠀⠀⠀⡎⠀⠀⢸⠀⢸⢸⢸⠀⡇⡇⠀⢸⢠⠃⡇⢀
⠀⠇⠀⠀⠀⢸⠀⡎⠀⠘⡄⠀⢸⠀⠀⠀⠘⡄⡜⢸⢸⠀⢣⠇⠀⢸⢸⠀⡇⢸
⠀⠀⠀⠀⠀⢸⠀⡇⠀⠀⡇⠀⡇⠀⠀⠀⠀⡇⡇⠸⡇⠀⢸⠀⠀⠀⣿⠀⢣⢸
⠀⠀⠀⠀⠀⢸⠀⡇⠀⠀⢸⢰⠁⠀⠀⠀⠀⡇⡇⠀⡇⠀⠀⠀⠀⠀⡇⠀⢸⢸
⠀⠀⠀⠀⠀⠸⣰⠁⠀⠀⠀⠇⠀⠀⠀⠀⠀⢳⠁⠀⠇⠀⠀⠀⠀⠀⠁⠀⢸⢸
⠀⠀⠀⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⡸
⠀⠀⠀⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⡇
⠀⠀⠀⠀⠀⠀⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡇
⠀⠀⠀⠀⠀⠀⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠃`

However, when added to a pytermgui widget it does not render correctly. 

Is there any easy way to overcome this?

Also -- thank you for sharing your library, it really is awesome.

**Comment by bczsalba** [comment]
Good day!

This is a known issue with the line-breaker algorithm, AFAIK it currently completely ignores `\n` characters. This will be fixed sometime post-1.0.0, until then you can add content line-by-line to a widget for a band-aid fix.

<img width="647" alt="Screenshot 2021-09-30 at 21 04 10" src="https://user-images.githubusercontent.com/66030897/135515424-ebab3bb0-6292-4421-8e55-58beb8906297.png">

All I'm doing here is unpacking a generator that goes through the plot split by linebreaks. Any unnamed argument to `Container` is considered a potential `Widget`, so the `auto()` method is run on it to convert it as such. A more verbose way would be this:

```python3
# `plot` is already defined above
import pytermgui as ptg

root = ptg.Container()
for line in plot.splitlines():
    root += ptg.Label(line)

root.print()
```

Thank you for your interest, I hope this module will be useful for you. If you create any public projects with it, you can always contact me for a shoutout in the `README`.

**Comment by Tbruno25** [comment]
Thank you for the temporary solution!

I spent an hour experimenting and think I'm going to hold off until the api / library is a bit more mature. I look forward to revisiting this in the near future!
